src/kaltura-player.ts

```diff
@@ -832,15 +836,17 @@ export class KalturaPlayer extends FakeEventTarget {
   }
 
   private _addAudioPreset(options: Partial<KalturaPlayerConfig>): void {
```

Comment:
rename to _addCustomPreset or Presets if this becomes a general use function

Reply:
done